dojo/1.11/dojo.d.ts

```diff
@@ -784,8 +784,8 @@ declare namespace dojo {
 	/* dojo/Evented */
 
 	interface Evented {
-		on(type: string | ExtensionEvent, listener: EventListener): Handle;
-		emit(type: string | ExtensionEvent, event: any): boolean;
+		on(type: string | ExtensionEvent, listener: EventListener | Function): Handle;
+		emit(type: string | ExtensionEvent, ...events: any[]): boolean;
```

Comment:
I'm not sure I agree with this definition change for emit... wouldn't this force event/events to always be an array when it's often just an object? Perhaps you need both call signatures?